textdb/textdb-sandbox/JinggangHelloWorld.java

```diff
@@ -1,6 +0,0 @@
-#include<iostream>
-public class JinggangHelloWorld{
-	public static void main(String[] args){
-		cout << "Hello World!" << endl;
```

Comment:
@chenlica @diaojinggang .DS_Store is a Mac specific attributes file that is generated by Finder in the Mac environment to store parameters such as order of files in a directory, etc. A convenient way to ensure this don't show up is to add it to a .gitignore file?